### DISCUSSION TITLE
Can You Link to Different Pages in Mermaid Charts?

### DISCUSSION THREAD

**Discussion by elizaodie** [GitbookIO/community#333] [0. General bugs and questions]
In some places, you can use nodes in Mermaid charts to link to other URLs/pages. Is this capability supported in GitBook? I tried it out but it doesn't appear to be working for me.

**Comment by rodrcastro** [comment]
Hey @elizaodie 👋 Sorry for the delay in the response here.

This is currently not possible. This requires a change in our Mermaid integration that might open a vector for malicious code to be injected into GitBook pages, which is something that we definitely don't want.

Our team will be investigating that option in the future, though. Thank you for the feedback!
